src/www/ui/api/Models/Upload.php

```diff
@@ -96,12 +96,12 @@ public function getJSON()
   public function getArray()
   {
     return [
-      "folderid"    => $this->folderId,
-      "foldername"  => $this->folderName,
+      "folderId"    => $this->folderId,
+      "folderName"  => $this->folderName,
       "id"          => $this->uploadId,
       "description" => $this->description,
-      "uploadname"  => $this->uploadName,
-      "uploaddate"  => $this->uploadDate,
+      "uploadName"  => $this->uploadName,
+      "uploadDate"  => $this->uploadDate,
       "assignee"    => $this->assignee,
       "hash"        => $this->hash->getArray()
```

Comment:
I would instead recommend to make the model version aware. See https://github.com/fossology/fossology/pull/2617/files#diff-f62f1d2b860c504c87d386cdfa79513c109848593f0afeb60a5dc76e4b98ff42 for reference.